jupyterhub/spawner.py

```diff
@@ -943,15 +943,14 @@ async def move_certs(self, paths):
         """Takes certificate paths and makes them available to the notebook server
 
         Arguments:
-            paths (dict): a list of paths for key, cert, and CA.
-            These paths will be resolvable and readable by the Hub process,
-            but not necessarily by the notebook server.
+            paths (dict): a list of paths for key, cert, and CA. These paths
+            will be resolvable and readable by the Hub process, but not
+            necessarily by the notebook server.
```

Comment:
```suggestion
            paths (dict): a list of paths for key, cert, and CA.
              These paths will be resolvable and readable by the Hub process,
              but not necessarily by the notebook server.
```

Reply:
Resolved in 0679586 (but with four spaces instead of two, hmm...)